dist/aws-sdk-react-native.js

```diff
@@ -37216,14 +37217,25 @@ return /******/ (function(modules) { // webpackBootstrap
 	    var dnsSuffix = regionUtil.getEndpointSuffix(accessPointArn.region);
 	    var useArnRegion = req.service.config.s3UseArnRegion;
 
-	    endpoint.hostname = [
-	      accessPointArn.accessPoint + '-' + accessPointArn.accountId + outpostsSuffix,
-	      serviceName + dualStackSuffix,
-	      useArnRegion ? accessPointArn.region : req.service.config.region,
-	      dnsSuffix
-	    ].join('.');
-	    endpoint.host = endpoint.hostname;
+	    if (isObjectLambdaArn) {
```

Comment:
Remove diffs in `dist/`, they are generated.

Reply:
done